grafana-plugin/src/components/Policy/NotificationPolicy.tsx

```diff
@@ -47,21 +43,29 @@ export interface NotificationPolicyProps {
 }
 
 export class NotificationPolicy extends React.Component<NotificationPolicyProps, any> {
+  private styles: ReturnType<typeof getStyles>;
+
+  constructor(props: NotificationPolicyProps) {
+    super(props);
+
+    this.styles = getStyles(props.theme);
```

Comment:
Have you tested if this part works when the theme is changed? Constructor will not be called unless the page reload happens afaik so it might stick to outdated theme

Reply:
That's a good point, let me try that

Reply:
Updated those 2 usecases, they're now part of componentDidUpdate lifecycle